src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyGraphOperation.cs

```diff
@@ -193,20 +277,60 @@ private string GetGraphVizNodeName(DependencyGraphNode node)
             return GetSimpleRepoName(node.RepoUri).Replace("-", "") + node.Commit;
         }
 
-        private void LogGraphViz(DependencyGraph graph)
+        private async void LogGraphViz(DependencyGraph graph)
```

Comment:
Async void is *BAD*. The caller cannot wait for it to complete, so it's a crap shot as to when, or even _if_, it will finish. And any exceptions it's throws are 100% unable to be responded to and get lost to the void.

Reply:
Ya should be async task

Reply:
It was non async Console.WriteLine before and got changed over to writing a file.  That explains why it was failing silently.  Why is there no warning for async void.